api/nodes/views.py

```diff
@@ -114,6 +116,14 @@
 from website.exceptions import NodeStateError
 from website.project import signals as project_signals
 
+# This is used to rethrow v1 exceptions as v2
+HTTP_CODE_MAP = {
+    400: ValidationError(detail='This provider has made a bad request.'),
+    401: NotAuthenticated,
+    403: PermissionDenied,
+    404: NotFound,
```

Comment:
Should these other errors be given similar messages to the 400 error?   Just to communicate that it's an issue with the provider.